CRM/Utils/Address.php

```diff
@@ -136,7 +136,6 @@ public static function format(
         'contact.birth_date' => $fields['birth_date'] ?? NULL,
         'contact.gender' => $fields['gender'] ?? NULL,
         'contact.is_opt_out' => $fields['is_opt_out'] ?? NULL,
-        'contact.preferred_mail_format' => $fields['preferred_mail_format'] ?? NULL,
```

Comment:
I'd be amazed if someone is using this for something like display_name format: `Dr. John Smith (Text)`

Reply:
@demeritcowboy you sell him short - he is a modern man ... `Dr. John Smith (Both)`

Reply:
hehe